crates/bevy_text/src/font_atlas.rs

```diff
@@ -118,3 +118,139 @@ impl core::fmt::Debug for FontAtlas {
             .finish()
     }
 }
+
+/// Adds the given subpixel-offset glyph to the given font atlases
+pub fn add_glyph_to_atlas(
```

Comment:
Maybe we can change the visibility to `pub(crate)`. I don't think anything is depending on this function externally.
```suggestion
pub(crate) fn add_glyph_to_atlas(
```